### PR TITLE
Update the unattended install test in accordance with the new VT runner

### PR DESCRIPTION
**Pull request by pevogam**
These are adaptations for the unattended install tests to run again with the new VT runner.

Signed-off-by: Plamen Dimitrov <plamen.dimitrov@intra2net.com>

### CODE REVIEW

File: virttest/shared/deps/run_autotest/kernel_install/kernelinstall.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def _kernel_install_src(self, base_tree, config=None, config_list=None,
             base_tree = os.path.join(self.bindir, base_tree)
         if not utils.is_url(config):
             config = os.path.join(self.bindir, config)
-        kernel = self.job.kernel(base_tree, self.outputdir)
+        kernel = self.job.kernel(base_tree, self.logdir)
```
Comment (richtja) [inline virttest/shared/deps/run_autotest/kernel_install/kernelinstall.py:86]: Hi @pevogam, the problem with `outputdir` is fixed by #3214. So I think this change is not necessary.

**Review by richtja — CHANGES_REQUESTED** [review]

File: virttest/shared/deps/run_autotest/kernel_install/kernelinstall.py (hunk 1 above)
Comment (pevogam) [reply virttest/shared/deps/run_autotest/kernel_install/kernelinstall.py:86]: Thanks for talking a look, this is just some fixes I collected that were lacking on master. Alright, I will take a look into your PR first and then see what is left here.

**Comment by pevogam** [comment]
@richtja Do you think the currently remaining diff is still a valid fix? If not I will close this and if yes I will rebase it.

**Comment by richtja** [comment]
@pevogam after the #3226 this is no more needed. Actually I think we can move the `TestUtils` functionality back to the `test.VirtTest` class, because now there isn't any other usage of it elsewhere.

**Comment by pevogam** [comment]
Alright, in this case I am closing this.

Regarding later refactoring of the VT runner, it is probably better to wait after the LTS release and perform regression fixes only.

**Pull request closed without merging**